### PR TITLE
[E2E] Add aggregate feature importance tests for responsibleaitoolbox-classification-model-assessment notebook

### PR DESCRIPTION
**Pull request by vinuthakaranth**
Adding e2e aggregate feature importance tests for responsibleaitoolbox-classification-model-assessment notebook.
Moving utils to parent folder so that it can be used by dashboard-e2e and widget e2e.

### CODE REVIEW

**Comment by vinuthakaranth** [comment]
I tried to mode utils folder under /apps but lead to lint error.

File: apps/widget-e2e/src/describer/modelAssessment/featureImportances/aggregateFeatureImportance/describeAggregateFeatureImportance.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+): void {
+  const datasetShape = modelAssessmentDatasets[name];
+  if (datasetShape.featureImportanceData?.noFeatureImportance) {
+    return;
+  }
+  describe(testName, () => {
+    before(() => {
+      const hosts = Cypress.env().hosts;
+      cy.task("log", hosts);
+      cy.visit(hosts[0].host);
```
Comment (xuke444) [inline apps/widget-e2e/src/describer/modelAssessment/featureImportances/aggregateFeatureImportance/describeAggregateFeatureImportance.ts:26]: I think this one would not work.
There are 3 notebooks started at same time, we probably need to based on file name and find coorposding host to visit here.

Comment (vinuthakaranth) [reply apps/widget-e2e/src/describer/modelAssessment/featureImportances/aggregateFeatureImportance/describeAggregateFeatureImportance.ts:26]: Thanks for the comment, I agree, will update the code.

File: apps/widget-e2e/src/describer/modelAssessment/featureImportances/aggregateFeatureImportance/describeGlobalExplanationChart.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  TElement extends IChartElement
+>(props: { chart: Chart<TElement>; dataShape: IModelAssessmentData }): void {
+  describe("Global explanation chart", () => {
+    it("should have y axis label", () => {
+      cy.get('#FeatureImportanceBar div[class*="rotatedVerticalBox"]').should(
+        "contain.text",
+        "Aggregate feature importance"
+      );
+    });
+    it("should have x axis label", () => {
```
Comment (imatiach-msft) [inline apps/widget-e2e/src/describer/modelAssessment/featureImportances/aggregateFeatureImportance/describeGlobalExplanationChart.ts:30]: so this all looks very similar to:

responsible-ai-widgets\apps\dashboard-e2e\src\describer\interpret\aggregateFeatureImportance\describeGlobalExplanationChart.ts

I am wondering if there is a better way to refactor this code, as it seems to be duplicating the other test code?  Or did I misunderstand?

Comment (imatiach-msft) [reply apps/widget-e2e/src/describer/modelAssessment/featureImportances/aggregateFeatureImportance/describeGlobalExplanationChart.ts:30]: Couldn't you just use the other file directly here?

File: apps/widget-e2e/src/util/BoxChart.ts [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+      throw new Error(
+        `box${idx} left: ${boxCoordinate.left} does not match left for mean: ${meanCoordinate.left} `
+      );
+    }
+    if (meanCoordinate && boxCoordinate.right !== meanCoordinate.right) {
+      throw new Error(
+        `box${idx} right: ${boxCoordinate.right} does not match right for mean: ${meanCoordinate.right} `
+      );
+    }
+    // if (boxCoordinate.bottom < meanCoordinate.mean) {
```
Comment (imatiach-msft) [inline apps/widget-e2e/src/util/BoxChart.ts:54]: remove commented out code

**Review by imatiach-msft — APPROVED** [review]

File: apps/widget-e2e/src/describer/modelAssessment/featureImportances/aggregateFeatureImportance/describeGlobalExplanationChart.ts (hunk 2 above)
Comment (imatiach-msft) [reply apps/widget-e2e/src/describer/modelAssessment/featureImportances/aggregateFeatureImportance/describeGlobalExplanationChart.ts:30]: oh, was this supposed to be a move and not a copy?  It looks like the old files were not removed.

Comment (vinuthakaranth) [reply apps/widget-e2e/src/describer/modelAssessment/featureImportances/aggregateFeatureImportance/describeGlobalExplanationChart.ts:30]: Yes, as for now it is copied and there are few changes in some files. And more will be added in future in similar directory structure. Ultimately files under dashboard-e2e/static dashboard tests will be removed once all real time notebook tests are in.

File: apps/widget-e2e/src/util/BoxChart.ts (hunk 3 above)
Comment (vinuthakaranth) [reply apps/widget-e2e/src/util/BoxChart.ts:54]: sure, will do in coming PR

**Pull request merged**